source/reference/compatibility.txt

```diff
@@ -202,6 +202,37 @@ in ``mongosh`` better align with the types used by the MongoDB Drivers.
    For more information on managing types, refer to the
    :manual:`schema validation overview </core/schema-validation>`.
 
+Undefined Values
+----------------
+
+The undefined BSON type is `deprecated <https://bsonspec.org/spec.html>`__ in
+:binary:`bin.mongosh`. If you insert a document with the undefined BSON type in
+:binary:`bin.mongosh`, your deployment stores a null value. To create undefined
```

Comment:
(question)
Do you think we could replace "stores a null value" with something a little less jargony, that retains the same meaning? Maybe add something about the impact to to document or to data when undefined is converted to null?